### DISCUSSION TITLE
Timeline for MFA feature

### DISCUSSION THREAD

**Discussion by thiner** [logto-io/logto#4713] [Q&A] [answered]
Logto.io is awesome! I like ti so much, and want to use it in my project. However, MFA is a required feature for user authentication service in my organization. You have put a MFA on the homepage as a coming soon feature, may I know when will it be available?

Thanks!

**Comment by xiaoyijun** [comment]
Hi @thiner , the MFA feature will be released in the early Nov. :)

**Comment by charIeszhao** [comment, marked as answer]
The MFA feature has been officially released. You can check the documentation for more details.
https://docs.logto.io/docs/recipes/multi-factor-auth/
